analyzer/cli/src/main/java/jp/co/ntt/oss/heapstats/cli/CliMain.java

```diff
@@ -28,21 +29,16 @@
  * 
  * @author Yasumasa Suenaga
  */
-public class CliMain {
+public class CliMain implements Thread.UncaughtExceptionHandler{
 
     /**
      * @param args the command line arguments
      */
     public static void main(String[] args) {
         Options options = new Options();
+        Thread.setDefaultUncaughtExceptionHandler(new CliMain());
         
-        try{
-            options.parse(args);
-        }
-        catch(Exception e){
-            options.printHelp();
-            System.exit(1);
-        }
+        options.parse(args);
```

Comment:
IMHO, I think that most users (who are not developers) want to read help messages instead of exception when they set wrong options.